src/common/rollbar.config.js

```diff
@@ -0,0 +1,60 @@
+import { rollbarAccessToken } from 'common/app.constants';
+import rollbar from 'rollbar-browser';
+import stacktrace from 'stacktrace-js';
+
+/* @ngInject */
+function rollbarConfig(envServiceProvider, $provide) {
+  // eslint-disable-next-line angular/window-service
+  if(window.__karma__ !== undefined) return; // Don't decorate $log in test env
+
+  let rollbarConfig = {
+    accessToken: rollbarAccessToken,
+    captureUncaught: true,
+    captureUnhandledRejections: true,
+    environment: envServiceProvider.get(),
+    enabled: !envServiceProvider.is('development'), // Disable rollbar in development environment
+    verbose: envServiceProvider.is('development') // Log rollbar errors to console in development environment
```

Comment:
This may be unwanted and just clutter up our development consoles. It would help us see which errors are getting sent to rollbar though. What do you guys think/prefer?